packages/async-rewriter2/AUTHORS

```diff
@@ -0,0 +1,8 @@
+Anna Herlihy <herlihyap@gmail.com>
+Durran Jordan <durran@gmail.com>
+Maurizio Casimirri <maurizio.cas@gmail.com>
+Massimiliano Marcon <me@marcon.me>
+Rhys Howell <rhys@rhysh@live.com>
+Irina Shestak <shestak.irina@gmail.com>
+Liudmila Kornilova <kornilova203@gmail.com>
+Anna Henningsen <anna@addaleax.net>
```

Comment:
Opened a ticket to also make this file autogenerated, like we do with the `THIRD_PARTY_NOTICED.md`: https://jira.mongodb.org/browse/MONGOSH-647